tasks/build-js

```diff
@@ -5,10 +5,14 @@
 const fs = require('fs');
 const browserify = require('browserify');
 const watchify = require('watchify');
+const outputDir = './web/build/';
```

Comment:
Why are we creating a new build directory in `/web` instead of using `/build`?

Reply:
at this point... at least a few tasks depend on everything being located in the web directory